src/util.rs

```diff
@@ -4,7 +4,7 @@ use std::path::{Path, PathBuf};
 use std::process::Command;
 use std::str;
 
-use color_eyre::{eyre, Result};
+use color_eyre::{eyre, eyre::bail, eyre::Context, Result};
```

Comment:
This is what I get running `fix.sh` in your branch
```suggestion
use color_eyre::{eyre, eyre::bail, Result};
```